backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -204,10 +213,38 @@ export class ContributionResolver {
 
     // all possible cases not to be true are thrown in this function
     validateContribution(creations, amount, creationDateObj)
+
+    const contributionMessage = ContributionMessage.create()
+    contributionMessage.contributionId = contributionId
+    contributionMessage.createdAt = contributionToUpdate.updatedAt
+      ? contributionToUpdate.updatedAt
+      : contributionToUpdate.createdAt
+    let changeMessage = ''
+    if (contributionToUpdate.memo !== memo) {
+      // TODO: Add message to changeMessage
+      changeMessage += 'Alter Memo: ' + contributionToUpdate.memo + '\n'
+    }
+    if (contributionToUpdate.amount.toFixed(6) !== amount.toFixed(6)) {
+      // TODO: Add amount to changeMessage
+      changeMessage += 'Alter Betrag: ' + contributionToUpdate.amount + '\n'
+    }
+    const isDateDifferent =
```

Comment:
why use a variable here? Do you reuse the value?

Reply:
I did it cause I tested every expression in a console log.... Maybe we can use it for the logging later on